package.json

```diff
@@ -108,5 +108,9 @@
     "yargs": "^13.2.2",
     "zone.js": "^0.9.1"
   },
-  "keywords": []
+  "keywords": [],
+  "dependencies": {
+    "gif-frames": "^1.0.1",
+    "gifshot": "^0.4.5"
```

Comment:
Just for info how big are these libraries?

Reply:
gifshot is about 24kb when minified:
![image](https://user-images.githubusercontent.com/19631367/66729810-a9586a80-ee02-11e9-908c-079365df09b8.png)

Reply:
gif-frames is kind of large at 122kb. I was using it because it provided the simplest interface for breaking apart GIF files, but I can look into other packages if managing bundle size is a priority. 

![image](https://user-images.githubusercontent.com/19631367/66729830-c3924880-ee02-11e9-80b8-df1db92e3149.png)

Reply:
That seems okay, thanks!

Reply:
Sorry, I didn't notice your second message, 122 kB is quite a lot although we use gzip so it is 35 kB, but please look into other packages and if you find something thats significantly smaller and also easy to implement then use it.

Reply:
@vojtechjelinek I did a bit of research and tried to get `[libgif-js](https://github.com/buzzfeed/libgif-js)` which is a smaller library to work, but I haven't been able to figure it out yet. any idea of other libraries for parsing through gifs on the frontend?

Reply:
I don't know any libs. But if you spend that much time and didn't find anything you can probably use the libs you used before.